apps/cookbook/src/app/examples/radio-example/examples/template-driven-forms.ts

```diff
@@ -45,46 +39,26 @@ const config = {
       font-style: italic;
     }`,
   ],
-  codeSnippet: `
-items = [
-  { title: 'Bacon', value: 1 },
-  { title: 'Salami', value: 2 },
-  { title: 'Tenderloin', value: 3 },
-  { title: 'Tongue', value: 4 },
-  { title: 'Drumstick', value: 5 },
+  codeSnippet: `items = [
+  'Bacon',
+  'Salami',
+  'Tenderloin',
+  'Tongue',
+  'Drumstick',
 ];
+
 selected = null;`,
 };
 
 @Component({
   selector: config.selector,
   template: config.template,
-  styles: [
-    `
-      :host {
-        display: flex;
-      }
-
-      kirby-form-field:not(:last-of-type) {
-        margin-right: 40px;
-      }
-    `,
-  ],
+  styles: [],
```

Comment:
Remove empty prop:
```suggestion
```
OR maybe apply margin-top to cookbook-example-configuration-wrapper as in reactive-forms example?